### PR TITLE
Search screens

### PR DESCRIPTION
**Pull request by girishsawant999**

### CODE REVIEW

File: www/src/components/containers/search/components/CountryList/index.tsx [hunk 1]
```diff
@@ -1,54 +1,48 @@
...
+                  height="89px"
+                  className="rounded-2xl"
+                />
+                <div className="ml-4">
+                  <Typography
+                    type="content"
+                    variant="h2"
+                    size="16"
+                    weight="extra-bold">
+                    <>{country.name}</>
```
Comment (Pranav2612000) [inline www/src/components/containers/search/components/CountryList/index.tsx:37]: Is the react fragment required ?

File: www/src/components/containers/search/components/SearchInputComp/index.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+function SearchInput({
+  searchQuery,
+  setsearchQuery,
+  onSearchQueryChange
+}: ISearchInput) {
+  return (
+    <div className="flex py-4 px-5 items-center w-full sticky bg-white top-0 transition-all">
+      <BackButton onClick={() => null} className="static" />
+      <form action="" className="w-full relative">
+        <input
```
Comment (Pranav2612000) [inline www/src/components/containers/search/components/SearchInputComp/index.tsx:21]: label missing. Verify for other inputs too

File: www/src/components/containers/search/index.tsx [hunk 3]
```diff
@@ -1,87 +1,54 @@
...
+    const resCountries = await fetchApi(`api/v1/ums/countries/list?name=${query}`);
+    if (resCountries && !resCountries.error && resCountries.response) {
+      let { response } = resCountries;
+      setResultCountries((response && response.data) || []);
+    }
+  };
 
-function SearchMain({}: ISearchMain) {
-  const [searchQuery, setsearchQuery] = useState<string>('');
+  const onSearchQueryChange = (event: { target: HTMLInputElement }) => {
```
Comment (Pranav2612000) [inline www/src/components/containers/search/index.tsx:19]: can we use useEffect here?

File: www/src/components/containers/search/components/CountryList/index.tsx (hunk 1 above)
Comment (girishsawant999) [reply www/src/components/containers/search/components/CountryList/index.tsx]: Removed the fragment

File: www/src/components/containers/search/components/SearchInputComp/index.tsx (hunk 2 above)
Comment (girishsawant999) [reply www/src/components/containers/search/components/SearchInputComp/index.tsx:20]: Added label

File: www/src/components/containers/search/index.tsx (hunk 3 above)
Comment (girishsawant999) [reply www/src/components/containers/search/index.tsx]: Implement with useEffect

**Pull request merged**